src/main/java/com/ericsson/ei/controller/info/InstanceInfoControllerImpl.java

```diff
@@ -39,8 +39,9 @@ public String parseInfo() {
         try {
             return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(istanceInfo);
         } catch (Exception e) {
-            LOGGER.error("Serialization has failed " + e.getMessage());
+            String errorMessage = "Failed to parse EI backend information. Error message:\n" + e.getMessage();
+            LOGGER.error(errorMessage);
+            return errorMessage;
         }
```

Comment:
This will cause that nothing will happen in EI Frontend,,, I guess just a white page._
I think this should be rewritten to use ResponseEntity. And update EI Frontend, if needed.

Reply:
in frontend we have white page and error message in jGrowl. I can print message in page instead of jGrowl

Reply:
ok,, jGrowl and sticky jGrowl is OK to use.
But how do Frontend JS code know if its http Error or not? when we don't have any HTTP Status code,, or am I missing something.

Reply:
Somehow JS know it. But if needed, I will use Response Entity.

Reply:
I think it is more correct to use ResponseEntity as response. Then we also get same way of handling response in every entrypoint.

Reply:
Done